implementations/rust/ockam/ockam_command/src/space/create.rs

```diff
@@ -22,7 +22,7 @@ const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt"
 )]
 pub struct CreateCommand {
     /// Name of the space - must be unique across all Ockam Orchestrator users.
-    #[arg(display_order = 1001, value_name = "SPACE_NAME", default_value_t = hex::encode(&random::<[u8;4]>()), hide_default_value = true, value_parser = validate_space_name)]
+    #[arg(display_order = 1001, value_name = "SPACE_NAME", default_value_t = hex::encode(&random::<[u8;4]>()), hide_default_value = true, value_parser = validate_space_name)] 
```

Comment:
I think you forgot to actually use the `random_name` function right? You need to update the `default_value_t` attribute.

Reply:
I have committed the change, can you please review it now